### ISSUE TITLE
Can't insert the horizontal ellipsis (…) with Alt+0133 code when Num Lock is off

### ISSUE THREAD

**Issue by bboogwin** [notepad-plus-plus/notepad-plus-plus#14710]
### I can't insert the horizontal ellipsis (…) with Alt+0133 code when Num Lock is off
In case Num Lock is off, when I try to insert the **…** character using its Alt+ code (**0133**) in the main editor field of Notepad++, the SOH character (invisible by default) is inserted instead (whose Alt code is in fact **01**, not 0133). For other applications, the ellipsis character is inserted correctly with the same combination. More than that, when I open some model window, like **Find** or **Plugins Admin**, and start typing in any text field, the ellipsis is inserted OK with the same Alt code!

### Steps to Reproduce the Issue
1. Open Notepad++
2. Make sure the Num Lock is off (e.g. numpad buttons do not enter digits or other characters when typed, but manage the cursor)
3. (optional) Enable **Show All Characters** mode with the corresponding button if you want to see the inserted SOH character
4. Press and hold left **Alt** button
5. Press and release sequentially **0, 1, 3, 3** numpad buttons
6. Release **Alt** button

### Expected Behavior
Ellipsis (…) character appears in the main editor window.

### Actual Behavior
SOH character appears instead; it is invisible unless you enabled the **Show All Characters** mode .

### Debug Information
Notepad++ v8.6.2   (64-bit)
Build time : Jan 14 2024 - 02:16:00
Path : C:\Program Files\Notepad++\notepad++.exe
Command Line : 
Admin mode : OFF
Local Conf mode : OFF
Cloud Config : OFF
OS Name : Windows 11 Home Single Language (64-bit)
OS Version : 22H2
OS Build : 22621.3007
Current ANSI codepage : 1252
Plugins : 
    ComparePlugin (2.0.2)
    mimeTools (3)
    NppConverter (4.5)
    NppExport (0.4)

**Comment by alankilborn** [comment]
I can't attempt to replicate the problem as I don't have a keyboard with a numeric keypad. :-(

---

> ...the SOH character (invisible by default) is inserted instead...

SOH (a "control character") would be visible by default.

Defaults in 8.6.2:

![image](https://github.com/notepad-plus-plus/notepad-plus-plus/assets/77065706/46956fb3-235e-4f75-bda5-46f5174ad8cf)

**Comment by bboogwin** [comment]
In my case, it looks like this (and I hardly could change it because I never knew about this menu before now =) but anyway…):
![image](https://github.com/notepad-plus-plus/notepad-plus-plus/assets/29553842/8c816b2b-8cde-42e2-a215-c32e32db75c1)

**Comment by alankilborn** [comment]
_Show Control Characters_ is sort of new.  It could be that the first version to have that defaulted to unchecked if you upgraded from a previous version, but I doubt that.

Maybe more likely is a pressing of the toolbar button that looks like a paragraph symbol (backwards `P` with double vertical stem).  This will toggle everything shown in the menus on (and then off again with another press).

**Comment by rdipardo** [comment]
This is reproducible anytime numlock mode is off. Turn it on, and each of the <kbd>0</kbd>, <kbd>1</kbd> and <kbd>3</kbd> numpad keys are sent; turn it off, and only the  <kbd>1</kbd> key is sent, so the input becomes `U+0001 Start of Heading`, or `SOH`.

@bboogwin, turning on numlock mode completely solves the problem. Is there any reason why numlock needs to be switched off?

If you're frequently entering non-ASCII characters, there's [a plugin](https://github.com/rdipardo/nppHTMLTag#usage) that will convert code points into character literals and *vice versa* (e.g. `\u2026` <-> `…`), regardless of keyboard state.

---

> I can't attempt to replicate the problem as I don't have a keyboard with a numeric keypad. :-(

You may not have a physical numpad, but look closely at the standard keys. If they have subscript digits in the same style as the function key, there is most likely a virtual numpad in there, e.g.,

![ka02R000000hLrCQAU_en_US](https://github.com/notepad-plus-plus/notepad-plus-plus/assets/59004801/7afa2313-57dd-4e69-a6e7-80bae5dcb456)

You can toggle the "virtual" numlock when <kbd>Fn</kbd> is held down; pressing <kbd>Fn</kbd> along with any of the "second function" keys will enter digits instead of the default key codes.

**Comment by alankilborn** [comment]
> You may not have a physical numpad, but

Yes, I have that, but I can't, with confidence, attempt to repro the OP's exact problem with that arrangement.  Meaning, that without "numlock" on, I can't get digit entry from those keys, or alt+... special character entry.  So...I figured I was better off saying I couldn't try to reproduce.  :-)
